src/test/java/org/prebid/server/bidder/eplanning/EplanningBidderTest.java

```diff
@@ -0,0 +1,363 @@
+                .containsExactly("http://eplanning.com/5a1ad71d2d53a0f5");
+    }
+
+    @Test
+    public void makeHttpRequestShouldReturnTwoHttpRequestsWhenTwoImpsHasDifferentExchangeIds() {
+        // given
+        final BidRequest bidRequest = BidRequest.builder()
+                .imp(asList(Imp.builder()
+                                .banner(Banner.builder().build())
+                                .ext(Json.mapper.valueToTree(ExtPrebid.of(null, ExtImpEplanning.of("exchangeId1"))))
+                                .build(),
+                        Imp.builder()
+                                .banner(Banner.builder().build())
+                                .ext(Json.mapper.valueToTree(ExtPrebid.of(null, ExtImpEplanning.of("exchangeId2"))))
+                                .build()))
+                .build();
+
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = eplanningBidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).hasSize(2)
+                .extracting(HttpRequest::getUri)
+                .containsExactly("http://eplanning.com/exchangeId2", "http://eplanning.com/exchangeId1");
+    }
+
+    @Test
+    public void makeHttpRequestShouldReturnOneHttpRequestForTowImpsWhenImpsHasSameSourceId() {
+        // given
+        final BidRequest bidRequest = BidRequest.builder()
+                .imp(asList(Imp.builder()
+                                .banner(Banner.builder().build())
+                                .ext(Json.mapper.valueToTree(ExtPrebid.of(null, ExtImpEplanning.of("exchangeId1"))))
+                                .build(),
+                        Imp.builder()
+                                .banner(Banner.builder().build())
+                                .ext(Json.mapper.valueToTree(ExtPrebid.of(null, ExtImpEplanning.of("exchangeId1"))))
+                                .build()))
+                .build();
+
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = eplanningBidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).hasSize(1).extracting(HttpRequest::getUri)
+                .containsExactly("http://eplanning.com/exchangeId1");
+    }
+
+    @Test
+    public void makeBidsShouldReturnBidWithoutErrors() throws JsonProcessingException {
+        // given
+        final String response = mapper.writeValueAsString(BidResponse.builder()
+                .seatbid(singletonList(SeatBid.builder()
+                        .bid(singletonList(Bid.builder().impid("impId").build()))
+                        .build()))
+                .build());
+        final BidRequest bidRequest = BidRequest.builder().imp(singletonList(Imp.builder().id("impId").build()))
+                .build();
+
+        final HttpCall httpCall = givenHttpCall(HttpResponseStatus.OK.code(), response);
```

Comment:
Can be generalized like `HttpCall<BidRequest> xxx` as well `givenHttpCall` method.